src/View/Helper/PaginatorHelper.php

```diff
@@ -1106,4 +1153,21 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Helper method to deprecate the 'model' option throughout PaginationHelper.
+     *
+     * @deprecated
+     * @param array $options Array of options
+     * @return array Array of options
+     */
+    protected function _deprecateModelOption(array $options)
+    {
+        if (isset($options['model'])) {
+            $options['scope'] = $options['model'];
```

Comment:
How would userland code break on a parent call for a method that doesn't exist yet?